tests-beta/android/MASVS-NETWORK/MASTG-TEST-0239.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Using low-level APIs (e.g. Socket) to set up a custom HTTP connection
+platform: android
+id: MASTG-TEST-0239
+type: [static]
+weakness: MASWE-0050
```

Comment:
TODO: add support for multiple weaknesses?
This could be also https://mas.owasp.org/MASWE/MASVS-NETWORK/MASWE-0049/